lib/locales/en.ts

```diff
@@ -841,6 +870,7 @@ export const locale: Locale = {
     passion_of_saint_john_the_baptist: 'The Passion of Saint John the Baptist',
     patrick_of_ireland_bishop: 'Saint Patrick, Bishop',
     patrick_of_ireland_bishop_patron_of_ireland: 'Saint Patrick, Bishop, Patron of Ireland',
+    paterne_of_avranches_bishop_and_scubilion_abbot: 'Saint Paterne, Bishop, and Scubilion, Abbot',
```

Comment:
I’d prefer using _Paternus_ instead of _Paterne_ ([src](https://maryourhelp.org/saints/saint-paternus-bishop-of-avranches.html)).

```suggestion
    paterne_of_avranches_bishop_and_scubilion_abbot: 'Saint Paternus, Bishop, and Scubilion, Abbot',
```